packages/jsii-pacmak/test/expected.jsii-calc/dotnet/Amazon.JSII.Tests.CalculatorPackageId/Amazon/JSII/Tests/CalculatorNamespace/Submodule/Child/InnerClass.cs

```diff
@@ -0,0 +1,44 @@
+using Amazon.JSII.Runtime.Deputy;
```

Comment:
Is this correct? I remember having to remove a bunch of these files in https://github.com/aws/jsii/commit/586d037520de8d0a46a581f4bd8a54e62218dfe1#diff-8fc71bcffd7b86efe8bc3e18f8e26d4d.